config_examples/README.md

```diff
@@ -0,0 +1,45 @@
+# Collector example configurations
+
+Here you can find a collection of example Collector configurations.
+
+## Samples
+
+- [Drop](drop.yaml)
```

Comment:
Some of these samples are not clear what they are for/goal. For now I just added the name as the file name, but we should try to improve them in follow ups.

Reply:
Agreed, let's tackle that as we implement tests for each.